docs/usage/batches_jobs.md

```diff
@@ -55,6 +57,16 @@ batch.add_jobs([job2, job3], wait=True)
 batch.close()
 ```
 
+You can assign multiple tags to your batches when creating them to help organize and retrieve them later.
```

Comment:
nit: This is debatable, but I would suggest moving this example out of this section which is really focused around batch creation. Tags's purpose is to conveniently retrieve a list of batches so I would move this in the `Get a list of batches` section.

I would also put in the same snippet an example which creates a couple batches with different/shared tags and then a get_batches call to retrieve some of them. Maybe also in that same snippet show the usage of the `set_tags` method as well. what do you think?